courses/task_seven/seven_test.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"testing"
+	"reflect"
+)
+
+func TestSum(t *testing.T) {
+    total := Sum(5, 5)
+    if total != 10 {
+		t.Errorf("Sum was incorrect, got: %d, want: %d.", total, 10)
+    }
+}
+
+func TestMove(t *testing.T) {
+	type test struct {
+		input Vehicle
+		distance float64
+        want  interface{}
+	}
+
+	autoMark := Auto{"audi", 50.0, 8.5, 75.0}
+	var tr Vehicle = &autoMark
+
+	tests := []test{
+        {input: tr, distance: 100.0, want: nil},
+        {input: tr, distance: 200.0, want: nil},
+        {input: tr, distance: 600.0, want: VehicleError("Too small fuel!!!")},
+    }
+
+	for _, ex := range tests {
+        got := ex.input.Move(ex.distance)
+        if !reflect.DeepEqual(ex.want, got) {
```

Comment:
`reflect` should be avoided when not required. Here it looks like an overkill. E.g. maps couldn't be compared with `==` operation. But floats can be. Also note a comment on `Move` func